notebooks/book2/17/gprDemoChangeHparams.ipynb

```diff
@@ -2,14 +2,152 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    B = B.conjugate()
</pre><p>Why do you need this? B is a real-valued matrix</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>